### PR TITLE
[DX-1437] catalogue --> catalog (PR 2)

### PR DESCRIPTION
**Pull request by Roeegg2**
### For internal users - Please add a Jira DX PR ticket to the subject!
<br>

---

### Preview Link

<br>

### Description

<br>

#### Screenshots (if appropriate)
<br>

## Checklist

- [x] I have added a **preview link** to the PR description.
- [x] I have [reviewed the guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING.md) for contributing to this repository.
- [x] I have [read the technical guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [x] Make sure you have started *your change* off *our latest `master`*.
- [x] I **labelled** the PR

### CODE REVIEW

**Comment by Roeegg2** [comment]
In this PR I did change PortalAPICatalogue and HTML example, but didn't change JSON examples and URIs

Let me know if there is anything that I should change

File: tyk-docs/content/tyk-developer-portal/tyk-portal-classic/curity-dcr.md [hunk 1]
```diff
@@ -138,7 +138,7 @@ Take note of the `Key Hash` and `Key ID` as they will be needed later.
 
 #### Publish the API to the Developer Portal
 
-The API and the Facade API are now configured and can used to publish the API to the Tyk Developer Portal. Navigate to **Portal Management** &rarr; **Catalogue**, then click **Add New API**. Give it a public name, ex. `OAuth Facade API` and select the `facade-policy`. 
+The API and the Facade API are now configured and can used to publish the API to the Tyk Developer Portal. Navigate to **Portal Management** &rarr; **Catalog**, then click **Add New API**. Give it a public name, ex. `OAuth Facade API` and select the `facade-policy`.
```
Comment (letzya) [inline tyk-docs/content/tyk-developer-portal/tyk-portal-classic/curity-dcr.md:141]:
```suggestion
The API and the Facade API are now configured and can used to publish the API to the Tyk Developer Portal. Navigate to **Portal Management** &rarr; **Catalogue**, then click **Add New API**. Give it a public name, ex. `OAuth Facade API` and select the `facade-policy`. 
```

File: tyk-docs/content/tyk-developer-portal/tyk-portal-classic/gluu-dcr.md [hunk 2]
```diff
@@ -80,7 +80,7 @@ After the policy is ready, switch back to the API settings and make sure that th
 
 {{< img src="/img/dcr/gluu/step_6.png" alt="Step 6" >}}
 
-Now you’re ready to add this API to the developer portal. Switch to the "Catalogue" section under "Portal Management" on the navigation menu. Click on "Add New API", set a name for it and select the newly created policy. For this example use "Gluu Policy":
+Now you’re ready to add this API to the developer portal. Switch to the "Catalog" section under "Portal Management" on the navigation menu. Click on "Add New API", set a name for it and select the newly created policy. For this example use "Gluu Policy":
```
Comment (letzya) [inline tyk-docs/content/tyk-developer-portal/tyk-portal-classic/gluu-dcr.md:83]:
```suggestion
Now you’re ready to add this API to the developer portal. Switch to the "Catalogue" section under "Portal Management" on the navigation menu. Click on "Add New API", set a name for it and select the newly created policy. For this example use "Gluu Policy":
```

File: tyk-docs/content/tyk-stack/tyk-developer-portal/enterprise-developer-portal/getting-started-with-enterprise-portal/publish-api-products-and-plans.md [hunk 3]
```diff
@@ -39,20 +39,20 @@ Follow these steps below how to publish an API Product to a catalog:
...
-| Catalogue display name   | This is the name that will be displayed in the external catalog.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
+| Catalog display name   | This is the name that will be displayed in the external catalog.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
```
Comment (letzya) [inline tyk-docs/content/tyk-stack/tyk-developer-portal/enterprise-developer-portal/getting-started-with-enterprise-portal/publish-api-products-and-plans.md:42]:
```suggestion
| Catalogue display name   | This is the name that will be displayed in the external catalog.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              |
```

File: tyk-docs/content/getting-started/tutorials/publish-api.md [hunk 4]
```diff
@@ -65,13 +65,13 @@ Save the policy by clicking **Create**.
 
 The API that you defined earlier is active and will work as you've seen in the previous tutorial, this time we want to use the Portal to generate a token for a named developer.
 
-Not all APIs are visible to the Portal, only the ones you tell it about, so from the **Catalogue** menu, select **Add API** then:
+Not all APIs are visible to the Portal, only the ones you tell it about, so from the **Catalog** menu, select **Add API** then:
```
Comment (letzya) [inline tyk-docs/content/getting-started/tutorials/publish-api.md:68]:
```suggestion
Not all APIs are visible to the Portal, only the ones you tell it about, so from the **Catalogue** menu, select **Add API** then:
```

File: tyk-docs/content/product-stack/tyk-enterprise-developer-portal/getting-started/customize-products-and-plans.md [hunk 5]
```diff
@@ -26,7 +26,7 @@ In this section, you will learn how to customise the visual appearance of API Pr
 ## Customize API Products
 To customise the visual appearance of API Products:
 1. Navigate to the **API Products** menu, select the product that you want to customise (the Payment API in example below). In this menu you can customise the following aspects of API Products:
-- **Catalogue display name**: This is the name that will be displayed in the external catalog.
+- **Catalog display name**: This is the name that will be displayed in the external catalog.
```
Comment (letzya) [inline tyk-docs/content/product-stack/tyk-enterprise-developer-portal/getting-started/customize-products-and-plans.md:29]:
```suggestion
- **Catalogue display name**: This is the name that will be displayed in the external catalog.
```

File: tyk-docs/content/product-stack/tyk-enterprise-developer-portal/getting-started/customize-products-and-plans.md [hunk 6]
```diff
@@ -41,9 +41,9 @@ From that page you can also manage [OAuth2.0 settings]({{< ref "/tyk-stack/tyk-d
 
 Customise plans
 1. To customise visual appearance of plans, open the **Plans** menu and select the plan you want to customise. You can customise the following settings:
-- **Catalogue display name**: The name of the catalog that will be displayed in the API Product page.
+- **Catalog display name**: The name of the catalog that will be displayed in the API Product page.
```
Comment (letzya) [inline tyk-docs/content/product-stack/tyk-enterprise-developer-portal/getting-started/customize-products-and-plans.md:44]:
```suggestion
- **Catalogue display name**: The name of the catalog that will be displayed in the API Product page.
```

File: tyk-docs/content/getting-started/tutorials/publish-api.md [hunk 7]
```diff
@@ -65,13 +65,13 @@ Save the policy by clicking **Create**.
...
 
 1.  Select your **Default** policy
 2.  Fill in the description fields
 3.  Ensure the **Enable this API** is selected
 
-Save the API Catalogue entry by clicking **Update**.
+Save the API Catalog entry by clicking **Update**.
 
 {{< img src="/img/2.10/portal_confirmation_on_prem.png" alt="Catalogue Entry" >}}
```
Comment (letzya) [inline tyk-docs/content/getting-started/tutorials/publish-api.md]: alt="Catalogue Entry" - change to catalog please

**Review by letzya — APPROVED** [review]

**Pull request merged**